patch-package
Binary files a/node_modules/react-navigation/src/navigators/.DS_Store and b/node_modules/react-navigation/src/navigators/.DS_Store differ
--- a/node_modules/react-navigation/src/views/withNavigationFocus.js
+++ b/node_modules/react-navigation/src/views/withNavigationFocus.js
@@ -13,7 +13,7 @@ export default function withNavigationFocus(Component) {
       super(props);
 
       this.state = {
-        isFocused: props.navigation ? props.navigation.isFocused() : false,
+        isFocused: false,
       };
     }
 
